config/routes.rb

```diff
@@ -3,5 +3,11 @@
     post "/auth", to: "auth#create"
     get "/auth", to: "auth#show"
   end
+
+
+  resources :user, only: [] do
+    resources :scoresheets, defaults: { format: :json }
```

Comment:
Let's only nest routes that do not use the scoresheet id - index and create make most sense here. Update, show, and delete can be separate. 

GET - /users/1/scoresheets (index)
POST - /users/1/scoresheets (create) 
GET - /scoresheets/1 (show) 
PATCH - /scoresheets/1 (update)
DELETE - /scoresheets/1 (delete)